gq/bigutil.go

```diff
@@ -0,0 +1,22 @@
+package gq
+
+import (
+	"math/big"
+
+	"filippo.io/bigmod"
+)
+
+// leaks only the size of x
+func modAsInt(x *bigmod.Modulus) *big.Int {
+	return new(big.Int).SetBytes(x.Nat().Bytes(x))
+}
+
+// leaks only the size of x
+func natAsInt(x *bigmod.Nat, m *bigmod.Modulus) *big.Int {
+	return new(big.Int).SetBytes(x.Bytes(m))
+}
+
+// leaks only the size of x
+func intAsNat(x *big.Int, m *bigmod.Modulus) (*bigmod.Nat, error) {
```

Comment:
I personally don't think this is necessary given that we're using the name exported from another package which is explained in taht package's documentation: 
![image](https://github.com/openpubkey/openpubkey/assets/11447044/eafcaa26-4457-4a25-baf9-d503d7a20d8f)

Reply:
I was just confused when I first read it and had to do a google search. I wanted to provide feedback on my confusion in case you found it helpful.

I think it is reasonable not to put a comment, so no action here is needed and you can resolve this.